### PR TITLE
[RW-9771][risk=no]  Remove API hotswapping

### PR DESCRIPTION
**Pull request by NehaBroad**
The developers are dissatisfied with the current API hotswapping functionality and are exploring a redesign. 

This effort is documented in a ticket at https://precisionmedicineinitiative.atlassian.net/browse/RW-6768. The first step in this process is to remove the current code and start anew. This pull request achieves that goal by removing the IncrementalHotSwapTask logic.

Description:

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

**Review by evrii — APPROVED** [review]
Ran locally and worked as expected. 

I'm not sure if this is expected behavior or not, but once the api starts running, I get a message that something is "97% EXECUTING" in the bottom of the terminal.
image

**Review by jmthibault79 — APPROVED** [review]
Local works for me.  I can confirm that changing a Java file while it's running... does nothing :)

@evrii this has been confusing us for a long time, but it's harmless.  https://precisionmedicineinitiative.atlassian.net/browse/RW-3446

**Review by dmohs — APPROVED** [review]
Nice.

File: api/libproject/devstart.rb [hunk 1]
```diff
@@ -183,11 +183,8 @@ def run_api_incremental()
     common.status "Starting API server..."
     # appengineStart must be run with the Gradle daemon or it will stop outputting logs as soon as
     # the application has finished starting.
-    common.run_inline "./gradlew --daemon appengineRun &"
+    common.run_inline "./gradlew --daemon appengineRun "
```
[the remark this answers is not in the document]
Comment (NehaBroad) [reply api/libproject/devstart.rb]: Done

**Pull request merged**